### ISSUE TITLE
Automatically notify all CCRP PIs/devs when a new update is released

### ISSUE THREAD

**Issue by mdsimpson42** [colouring-cities/colouring-core#1234]
(I'm not sure what the best solution is, so I'm creating this issue so we've got somewhere to discuss it. I will look into it when I get a chance, but I'm open to suggestions.)

We need a way to notify the other CCRP partners when we've released a (major) update to the Core Platform, so that they know to sync their fork with the upstream changes.

Questions:

- Is there a way to do this automatically when a pull request is merged onto main?
- Could it be done via an announcement on [Discussions](https://github.com/orgs/colouring-cities/discussions) (at Organisation level)?
- Is it possible to tag a [team](https://github.com/orgs/colouring-cities/teams), like Everyone or CCRP Partners, so that they get a notification?

**Comment by tomalrussell** [comment]
Just testing - @colouring-cities/everyone do you receive this as a notification? What do you think about GitHub discussions as a channel for this kind of announcement?

As another option, I'd suggest considering a mailing list for this kind of thing - e.g. Apache projects run mailing lists alongside other ways of keeping in touch (see [arrow](https://arrow.apache.org/community/) for example)

**Comment by mdsimpson42** [comment]
I got an email about it, but it was a reply to my issue! 

I had thought about doing that. I've made a few announcements before. But the question is how to notify people so they see them without having to go to the Discussions page on the organisation.

A mailing list could be a good option. I guess we'd have to get people to sign up to it, so we have consent for GDPR. We could do a monthly newsletter or something too. I'll discuss it with Polly.
